components/GovernancePower/GovernancePowerForRole.tsx

```diff
@@ -45,49 +71,50 @@ export default function GovernancePowerForRole({
       <div className="animate-pulse bg-bkg-1 col-span-1 h-[76px] rounded-lg" />
     )
   }
-
   return (
     <>
-      {role === 'community' ? (
-        kind === 'vanilla' ? (
-          <div>
-            <VanillaVotingPower role="community" {...props} />
-            <Deposit role="community" />
-          </div>
-        ) : kind === 'VSR' ? (
-          didWithdrawFromVanillaSetup ? (
-            <LockedCommunityVotingPower />
-          ) : (
-            //TODO make a better generic little prompt for when a plugin is used but there are still tokens in vanilla
-            <>
-              <VanillaVotingPower role="community" {...props} />
-              <div className="flex flex-col gap-2">
-                <div>
-                  <small className="flex items-center mt-3 text-xs">
-                    <ExclamationIcon className="w-5 h-5 mr-2"></ExclamationIcon>
-                    Please withdraw your tokens and deposit again to get
-                    governance power
-                  </small>
-                </div>
-                <div>
-                  <VanillaWithdrawTokensButton role={role} />
-                </div>
+      {role === 'community' ? <>
+        {isVanilla && (
+              <div>
+                <VanillaVotingPower role="community" {...props} />
+                <Deposit role="community"/>
+              </div>
+          )}
+            {usesVSR && (
```

Comment:
This is a bit of a temporary solution until the third milestone (see my [demo ](https://www.loom.com/share/a588015b64a340bbb0b8f9ba4b9bd96f?sid=c5ed7d7f-122c-4d0b-83d2-7a37ba9d4003). However, the third milestone still focuses on "Vanilla + QV" in order to avoid scope creep.

Our solution in milestone 'works' for VSR + QV DAOs, by showing the VSR power (without using the word "vote") and then the true vote under the QV card (see below) but I can't say it is an all-encompassing solution. Finding a modular UI solution may not be the way to go here, and we should just go back to a simple card, with all the details hidden inside the My Governance Power UI.

In the absence of demand for a VSR + QV DAO I think we can punt on this decision, but if it is seen as a blocker for milestone 3 then we can make that change.

![image](https://github.com/solana-labs/governance-ui/assets/5008805/134f2393-43d2-4968-89ee-0ca7a3951294)